### ISSUE TITLE
If no stable release found consider Github?

### ISSUE THREAD

**Issue by indigoxela** [backdrop-contrib/backdrop_upgrade_status#8]
That's a useful tool!

A screenshot:
![screenshot-output](https://user-images.githubusercontent.com/842066/99147328-62314e80-2680-11eb-8988-277c40f4faf4.png)

Which leads me to a question/idea...

If there's no stable release yet it might be useful to check for an ongoing port on Github.
Both, i18n and l10n_update have no stable release, but both projects already exist in backdrop-contrib. If people only rely on the report they get, they might miss that someone's already working or at least has started working on a port. And that they could contribute to it.

Or does this exceed the purpose of this module?

**Comment by herbdool** [comment]
I think it's fair to include. Feel like creating a PR?

**Comment by herbdool** [comment]
I think it's first checking updates.backdrop.org for release and project info. In this case we might need a fallback which would check github to see if a page exists with the same project name. Or put in a hardcoded array to check.

**Comment by jenlampton** [comment]
Interesting, on my local I'm seeing the github projects (without releases), in yellow, as "dev versions" and the ones on b.org (with releases) in green, as "official releases". I wonder if there's an issue with connecting to github from some locations, that makes the dev versions not work consistently?

**Comment by jenlampton** [comment]
I've confirmed that this feature already exists: 

<img width="1537" alt="Screen Shot 2021-04-15 at 1 12 29 PM" src="https://user-images.githubusercontent.com/397895/114932401-4e834300-9dec-11eb-8236-1c53d4380c67.png">

@indigoxela can you update to the latest version and see if you are still having the problem?

**Comment by indigoxela** [comment]
@jenlampton in the meantime i18n does have a release (alpha), that's why it appears as available in the list. Modules without release still don't show up. Leaflet and Geophp do exist on Github, but have no release yet. Tested with version 7.x-1.1.

![upgrade-status-dev](https://user-images.githubusercontent.com/842066/114978218-3ad9ea00-9e89-11eb-988e-9bdd779b3e0f.png)

That's no bug, it's a feature request.

**Comment by jenlampton** [comment]
> in the meantime i18n does have a release (alpha)

Ah, right! nice :) 

> Modules without release still don't show up

Okay, I'm seeing this now too. Sorry for the hasty assumption :)

**Comment by jenlampton** [comment]
Alright, I've committed a first-go of this to the repo. I discovered the [GitHub API](https://docs.github.com/en/rest/reference/repos) is rate-limited for anonymous requests, so I've switched to checking the url for a 404 to see if the project exists in `backdrop-contrib` for the time being. I left all the github api code in there (commented out) so that it should be able to set up as soon as we figure out how to properly use the API :) 

I'll leave this issue open until we can add that, but at least people can see projects that are in development now.
